### ISSUE TITLE
--silent is not silent

### ISSUE THREAD

**Issue by danil-smirnov** [department-stockholm/directory-to-s3#3]
When run with npx, I'm getting 'DONE!' even if put -s/--silent switch.

**Comment by slaskis** [comment]
You're right, seems like I've missed to use that option. Feel like making a PR?

**Comment by danil-smirnov** [comment]
Well, `> /dev/null` works just great, so not feeling a lot of motivation to contribute...
